src/ucp/wireup/select.c

```diff
@@ -1069,9 +1078,11 @@ static void ucp_wireup_criteria_init(ucp_wireup_criteria_t *criteria)
 /**
  * Check whether emulation over AM is allowed for RMA/AMO lanes
  */
-static int ucp_wireup_allow_am_emulation_layer(unsigned ep_init_flags)
+static int ucp_wireup_allow_am_emulation_layer(ucp_context_h context,
+                                               unsigned ep_init_flags)
 {
-    return !(ep_init_flags & UCP_EP_INIT_FLAG_MEM_TYPE);
+    return !(ep_init_flags & UCP_EP_INIT_FLAG_MEM_TYPE) &&
+           !(context->config.features & UCP_FEATURE_EXPORTED_MEMH);
```

Comment:
why? don't we support exported for `rma_bw` lanes only for now?

Reply:
we should not allow AM emulation if EXPORTED memory handle feature was requested

Reply:
because if cross-gvmi memh will be passed and memory from HOST, it can't be used

Reply:
we do not support user memh for AMO - this use case is invalid

Reply:
the question was why is emulation needed for rma_bw lanes
also you added and error to AMO function if memh passed to it. Why do we need to disable AMO emulation?